website/docs/tutorial/1-setting-up.md

```diff
@@ -1,18 +1,16 @@
 ---
-title: Setting up
+title: Get started with dbt
 id: setting-up
+description: "Create your first dbt project using a SQL query."
```

Comment:
curious: Is this metadata for the page for search results?  Happy to keep an eye out for this on future pages.

Reply:
@coapacetic Yep, this supposedly will help with Google searches.